docs/topics/Gateway.md

```diff
@@ -336,21 +333,34 @@ Bots in under 100 guilds can enable these intents in the bot tab of the develope
 
-To specify these intents in your `IDENTIFY` payload, you must visit your application page in the Developer Portal and enable the toggle for each Privileged Intent that you wish to use. If your bot qualifies for [verification](https://dis.gd/bot-verification), you must first [verify your bot](https://support.discord.com/hc/en-us/articles/360040720412-Bot-Verification-and-Data-Whitelisting) and request access to these intents during the verification process. If your bot is already verified and you need to request additional privileged intents, [contact support](https://dis.gd/support).
+To specify privileged intents in your `IDENTIFY` payload, you must enable the priviledged intents your app needs. To toggle priviledged intents, navigate to your app's settings in the Developer Portal. Click on the **Bot** page, and under the "Privileged Gateway Intents" section you can enable the toggle for each intent your app needs. Verified apps can only use privileged intents once they've been approved for them.
 
-Events under the `GUILD_PRESENCES` and `GUILD_MEMBERS` intents are turned **off by default on all API versions**. If you are using **API v6**, you will receive those events if you are authorized to receive them and have enabled the intents in the Developer Portal. You do not need to use Intents on API v6 to receive these events; you just need to enable the flags.
+> info
+> Unverified apps can use privileged intents without approval, but still must enable them in their app's settings. If the app's verification status changes, it will then have to apply for the privileged intent(s).
 
-If you are using **API v8** or above, Intents are mandatory and must be specified when identifying.
+Events under the `GUILD_PRESENCES` and `GUILD_MEMBERS` intents are turned **off by default on all API versions**. If you are using **API v6**, you will receive those events if you are authorized to receive them and have enabled the intents in the Developer Portal. You do not need to use Intents on API v6 to receive these events; you just need to enable the flags. If you are using **API v8** or above, Intents are mandatory and must be specified when identifying.
+
+In addition to the gateway restrictions described here, Discord's REST API is also affected by Privileged Intents. For example, to use the [List Guild Members](#DOCS_RESOURCES_GUILD/list-guild-members) endpoint, you must have the `GUILD_MEMBERS` intent enabled for your application. This behavior is independent of whether the intent is set during `IDENTIFY`.
+
+#### Message Content Intent
+
+`MESSAGE_CONTENT (1 << 15)` is a unique privileged intent that isn't directly associated with any Gateway events. Instead, access to `MESSAGE_CONTENT` permits your app to receive message content data across the APIs, including (but not limited to) the `content`, `embeds`, `attachments`, and `components` fields.
```

Comment:
> including (but not limited to) the `content`, `embeds`, `attachments`, and `components` fields

according to everything else this is an exhaustive list; if it isn't then the full list should be published, instead of a partial list.

```suggestion
`MESSAGE_CONTENT (1 << 15)` is a unique privileged intent that isn't directly associated with any Gateway events. Instead, access to `MESSAGE_CONTENT` permits your app to receive message content data across the APIs, specifically the `content`, `embeds`, `attachments`, and `components` fields.
```

Reply:
there's `matched_content` as well: https://discord.com/developers/docs/topics/gateway#auto-moderation-action-execution-auto-moderation-action-execution-event-fields

i'm not sure we just want to be listing out random fields as an exhaustive list here—they probably make more sense in the references for the specific responses.

Reply:
maybe something like this?
```suggestion
`MESSAGE_CONTENT (1 << 15)` is a unique privileged intent that isn't directly associated with any Gateway events. Instead, access to `MESSAGE_CONTENT` permits your app to receive message content data across the APIs. For the message object, this applies to the `content`, `embeds`, `attachments`, and `components` fields; for other objects, the affected fields (if any) are listed in that object's documentation.
```

Reply:
Updated this a bit—lmk if confusing. I can take a deeper pass after I fold this into the updated Gateway documentation if it still needs some work